index.html

```diff
@@ -4253,9 +4100,175 @@ <h3>Parsing content</h3>
           return |record|.
         </li>
       </ol>  <!-- parsing NDEF unknown record -->
-    </p>
+    </div>
   </section>
 </section>
+</section>
+
+<!-- - - - - - - - - - - - - Security and Privacy - - - - - - - - - - - - - -->
+<section> <h2 id="security">Security and Privacy</h2>
+  <p>
+    The trust model, attacker model, threat model and possible mitigation
+    proposals for Web NFC are presented in the
+    <a href="http://w3c.github.io/web-nfc/security-privacy.html">
+    Security and Privacy</a> document. This section presents the chosen
+    security and privacy model through normative requirements to
+    implementations.
+  </p>
+
+  <section> <h3>Chain of trust</h3>
+  <p>
+    Web pages using Web NFC are not trusted.
```

Comment:
considered trusted

Reply:
Content of Security section will be updated in a future PR.